label_studio/data_import/api.py

```diff
@@ -459,8 +460,52 @@ def get(self, *args, **kwargs):
         file = file_upload.file
         if file.storage.exists(file.name):
             content_type, encoding = mimetypes.guess_type(str(file.name))
-            if content_type not in ['image/jpeg', 'image/gif', 'image/png', 'application/pdf']:
-                content_type = 'text/plain'
+            content_type = content_type or 'application/octet-stream'
+            mime_type_filter = [
+                    'image/svg+xml',
+                    # 'application/xhtml+xml',
+                    # 'text/html',
+                    # 'text/javascript'
+            ]
+            if content_type in mime_type_filter:
+                return RangedFileResponse(
+                        request, self._allowlist_svg(file),
+                        content_type=content_type)
+
             return RangedFileResponse(request, file.open(mode='rb'), content_type=content_type)
+
         else:
             return Response(status=status.HTTP_404_NOT_FOUND)
+
+    def _allowlist_svg(self, dirty_file):
+        """Define allowed tags for SVG files"""
+        from lxml.html import clean
+        import tempfile
+
+        allow_tags = [
+                'circle',
+                'ellipse',
+                'line',
+                'path',
+                'polygon',
+                'polyline',
+                'rect'
+        ]
+
+        cleaner = clean.Cleaner(
+                allow_tags=allow_tags,
+                remove_unknown_tags=False
+        )
+
+        fd_dirty = open(dirty_file.path, 'r')
```

Comment:
Better to write 
```
with open(dirty_file.path) as fd_dirty:
   ...
```

Reply:
Maybe we can just raise an exception here? and say to users that their SVGs are not secure and we can import them?

Reply:
sure, if that is what you want